Packs/PrismaCloud/ReleaseNotes/2_0_6.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Prisma Cloud (RedLock)
+- Change get-rql-response command to the `/search` endpoint instead of `/search/config` endpoint
+- Change the location of the timeRange model to the correct location.
```

Comment:
```suggestion
- Change ***redlock-get-rql-response*** command to use `/search` API instead of `/search/config` API.
- Fixed an issue in ***redlock-get-rql-response*** command where the **timeRange** wasn't calculated correctly.
```